integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ItMiiSampleHelper.java

```diff
@@ -0,0 +1,376 @@
+
+      // run JRF use cases irrespective of WLS use cases fail/pass
+      previousTestSuccessful = true;
+      execTestScriptAndAssertSuccess(domainTypeName, "-db,-rcu", "DB/RCU creation failed");
+    }
+
+    execTestScriptAndAssertSuccess(
+        domainTypeName,
+        "-initial-image,-check-image-and-push,-initial-main",
+        "Initial use case failed"
+    );
+  }
+
+  /**
+   * Test MII sample WLS or JRF update1 use case.
+   */
+  public static void callUpdateUseCase(String args,
+                                       String errString) {
+    if (args.contains("update3")) {
+      String imageName = (domainTypeName.equals(DomainType.WLS))
+          ? MII_SAMPLE_WLS_IMAGE_NAME_V2 : MII_SAMPLE_JRF_IMAGE_NAME_V2;
+      envMap.put("MODEL_IMAGE_NAME", imageName);
+    }
+
+    execTestScriptAndAssertSuccess(domainTypeName, args, errString);
+  }
+
+  /**
+   * Delete DB deployment for FMW test cases and Uninstall traefik.
+   */
+  public static void tearDownAll() {
+    logger = getLogger();
+    // uninstall traefik
+    if (traefikNamespace != null) {
+      logger.info("Uninstall traefik");
+      Command.withParams(new CommandParams()
+          .command("helm uninstall traefik-operator -n " + traefikNamespace)
+          .redirect(true)).execute();
+    }
+
+    // db cleanup or deletion
+    if (domainTypeName.equals(DomainType.JRF) && envMap != null) {
+      logger.info("Running samples DB cleanup");
+      Command.withParams(new CommandParams()
+          .command(MII_SAMPLES_SCRIPT + " -precleandb")
+          .env(envMap)
+          .redirect(true)).execute();
+    }
+  }
+
+
+  /**
+   * Set env variables map.
+   * @param newEnvMap a map contains env variables
+   */
+  public static void setEnvMap(Map<String, String> newEnvMap) {
+    ItMiiSampleHelper.envMap = newEnvMap;
+  }
+
+  /**
+   * Set domain type.
+   * @param domainTypeName domain type name
+   */
+  public static void setDomainType(DomainType domainTypeName) {
+    ItMiiSampleHelper.domainTypeName = domainTypeName;
+  }
+
+  /**
+   * Set image type.
+   * @param imageTypeName image type names
+   */
+  public static void setImageType(ImageType imageTypeName) {
+    ItMiiSampleHelper.imageTypeName = imageTypeName;
+  }
+
+  /**
+   * Get env variables map.
+   * @return a map containing env variables
+   */
+  public static Map<String, String> getEnvMap() {
+    return ItMiiSampleHelper.envMap;
+  }
+
+  /**
+   * Get domain type.
+   * @return domain type name
+   */
+  public static DomainType getDomainType() {
+    return domainTypeName;
+  }
+
+  /**
+   * Get image type.
+   * @return  image type names
+   */
+  public static ImageType getImageType() {
+    return ItMiiSampleHelper.imageTypeName;
+  }
+
+  /**
+   * Check whether to use auxiliary image or not.
+   * @return  true to test auxiliary image and false to test non-auxiliary image
+   */
+  public static boolean auxiliaryImageEnabled() {
```

Comment:
minor: This method can be removed and inlined - it's only called from one place

Reply:
removed